BallisticTables.json

```diff
@@ -0,0 +1,26 @@
+{"82mm_a3": [
+    {"charge": 0, "table": [
```

Comment:
lookup could be quicker by making the charge and range the key I think(not 100% sure how javascript deals with key iteration but I think that should be fine).
example:
```
{"82mm_a3": [
    {0: [
        100 : {qd blablabla},
        150: {more blablablabla}
        ]
   }
}
```
It makes the lookup code easier to write as you can directly lookup a key instead of iterate through items to find values.
lookup code being for example `rangeTables["82mm_a3"][0][150]`, no loops or anything. This also means that to find a range you can round your distance to target up and down to the nearest 50s/100s and perform a direct lookup with that number. No more loops!

The only downside to this is that we are making the rangetables a little less intuitive, though I don't think this is something we need to worry about.